lib/rules/headers/dl.js

```diff
@@ -129,6 +129,16 @@ export async function check(sr, done) {
                     )
                         sr.error(self, 'this-date');
                 } else sr.warning(self, 'no-date');
+
+                // Require new /TR shortnames to use lowercase. No RY published yet, so this rule applies to all RYs.
+                const needLowercase =
```

Comment:
You probably don't need to check each status. Since you are already relying on `isFP`, the only cases that would require the check are:
1. `isFP() === true`
2. shortname change (see how `isFP` is checking for such change)